build.gradle.kts

```diff
@@ -57,6 +59,16 @@ tasks.withType<Test>().configureEach {
     )
 }
 
+tasks.withType<KotlinJsTest>().configureEach {
+    environment("EnvironmentKey1", "EnvironmentValue1")
+    environment("EnvironmentEmpty", "")
+}
+
+tasks.withType<KotlinNativeTest>().configureEach {
+    environment("EnvironmentKey1", "EnvironmentValue1")
+    environment("EnvironmentEmpty", "")
+}
```

Comment:
Doesn't seem to work.

Reply:
I also had some issues with setting env variables that are visible tests. Maybe this will help:

https://github.com/OptimumCode/json-schema-validator/blob/main/test-suites/build.gradle.kts#L137

Reply:
Oh, there's a separate config for simulator-based tests... Thanks, will try it out!

Reply:
Works, thanks!